src/util.js

```diff
@@ -185,3 +185,32 @@ export function findClosestRecursively(element, selector) {
     )
   }
 }
+
+export function elementIsFocusable(element) {
+  const inertDisabledOrHidden = "[inert], :disabled, [hidden], details:not([open]), dialog:not([open])"
+
+  return !!element && element.closest(inertDisabledOrHidden) == null && typeof element.focus == "function"
+}
+
+export function queryAutofocusableElement(elementOrDocumentFragment) {
+  for (const element of elementOrDocumentFragment.querySelectorAll("[autofocus]")) {
+    if (elementIsFocusable(element)) return element
+    else continue
+  }
+
+  return null
```

Comment:
```suggestion
Array.from(elementOrDocumentFragment.querySelectorAll("[autofocus]")).find(e => 
  elementIsFocusable(e))
```

Reply:
Since `elementIsFocusable` accepts an element argument, we can omit the block and pass the function itself!

```js
return Array.from(elementOrDocumentFragment.querySelectorAll("[autofocus]")).find(elementIsFocusable)
```